### PR TITLE
console: fix clear text issue on RawSQL page

### PR DESCRIPTION
**Pull request by kolharsam**
### Description

- Cleans up the conditions and the actions that are used to control the behaviour of the migration checkbox and statement timeout box
- Fixes a minor issue on the `RawSQL` page on deleting/removing/updating any text from the editor.

### Affected components

- [x] Console

### Related Issues

https://github.com/orgs/hasura/projects/106#card-41469170

### CODE REVIEW

File: console/src/components/Services/Data/RawSQL/RawSQL.js [hunk 1]
```diff
@@ -107,7 +107,7 @@ const RawSQL = ({
 
   const submitSQL = () => {
     // set SQL to LS
-    localStorage.setItem(LS_RAW_SQL_SQL, sql);
+    localStorage.setItem(LS_RAW_SQL_SQL, sqlText);
```
Comment (soorajshankar) [inline console/src/components/Services/Data/RawSQL/RawSQL.js:110]: can we use the utility here?

Comment (soorajshankar) [reply console/src/components/Services/Data/RawSQL/RawSQL.js:110]: This you may ignore as we discussed, this is part of the other PR

**Comment by kolharsam** [comment]
@soorajshankar I've fixed that issue that you noticed. The `Run` is now disabled if the textarea is empty.

**Pull request merged**